R/radarchart.R

```diff
@@ -96,10 +97,13 @@ Radar <- function(x,
                     data.label.font.family = global.font.family,
                     data.label.font.size = 10,
                     data.label.font.color = global.font.color,
+                    data.label.font.autocolor = FALSE,
                     data.label.format = "",
                     data.label.prefix = "",
                     data.label.suffix = "",
                     data.label.values.only = FALSE,
+                    average.series = NULL,
+                    average.color = rgb(230, 230, 230, maxColorValue = 255),
```

Comment:
They actually are documented because there is a parameter of the same name in Column which it inherits documentation from. But I realised the description is not applicable to Radar charts. Updated in 9188fcc